lib/Ama/Model/Votes.pm

```diff
@@ -30,4 +37,18 @@ sub uncast {
   $@ ? { error => $@ } : $results;
 }
 
+sub _autoRemove {
```

Comment:
I like the word "remove" because it's the term we use when deleting in questions and comments. I liked autoRemove because the program does it automatically and it is not dependent on the owner or permissions of the user voting on it. In hindsight, I can see where it's a bit strange.